includes/DefaultSettings.php

```diff
@@ -3311,9 +3311,10 @@
 /**
  * Characters to prevent during new account creations.
  * This is used in a regular expression character class during
- * registration (regex metacharacters like / are escaped).
+ * registration.
+ * Wikia: Added filtering circled latin characters
  */
-$wgInvalidUsernameCharacters = '@';
+$wgInvalidUsernameCharacters = '/@|[\x{24B6}-\x{24E9}]|[\x{1F150}-\x{1F169}]/u';
```

Comment:
This variable name is confusing now. It tells me that forbidden characters are listed but I found regex pattern.